src/index.js

```diff
@@ -0,0 +1,147 @@
+      currentTemp = Number(response.data.main.temp);
+      count = Math.floor((currentTemp - 273.15) * 1.8) + 32;
+      totalCount.innerText = count;
+      console.log('success in find temperature!', response.data.main.temp);
+    })
+    .catch((error) => {
+      console.log('error in weather!');
+    });
+};
+
+// --------------------------------------wave5 Selecting the Sky-------------------
+const selectElement = document.getElementById('slide');
+const skyPicture = document.getElementById('skyPicture');
+
+const changeselectionSky = (value) => {
+  if (value === 'Sunny') {
+    skyPicture.textContent = '☁️ ☁️ ☁️ ☀️ ☁️ ☁️';
+  } else if (value === 'Cloudy') {
+    skyPicture.textContent = '☁️☁️ ☁️ ☁️☁️ ☁️ 🌤 ☁️ ☁️☁️';
+  } else if (value === 'Rainy') {
+    skyPicture.textContent = '🌧🌈⛈🌧🌧💧⛈🌧🌦🌧💧🌧🌧';
+  } else if (value === 'Snowy') {
+    skyPicture.textContent = '🌨❄️🌨🌨❄️❄️🌨❄️🌨❄️❄️🌨🌨';
+  }
+};
+selectElement.addEventListener('change', (event) => {
+  // console.log(event.target);
+  const value = event.target.value;
+  // console.log(event.target.value);
+  changeselectionSky(value);
+});
+// --------------------------------wave2  Increase and Decrease Temperature----------------------
+
+// Select increment and decrement buttons
+const increaseElement = document.getElementById('up');
+const decreaseElement = document.getElementById('down');
+// Select total count
+const totalCount = document.getElementById('totalCount');
+// Variable to track count
+// let count = 49;
+// Display initial count value
+// totalCount.innerHTML = count;
+// Function to increment/decrement count
+
+// Function to increment/decrement count
+const handleIncrease = () => {
+  count++;
+  totalCount.innerHTML = count;
+  checkTemperatureRange(Number(totalCount.innerText));
+};
+
+const handleDecrease = () => {
+  count--;
+  totalCount.innerHTML = count;
+  checkTemperatureRange(Number(totalCount.innerText));
+};
+
+// Add click event to buttons
+increaseElement.addEventListener('click', handleIncrease);
+decreaseElement.addEventListener('click', handleDecrease);
+// Temperature Ranges Change
+const gardenPicture = document.getElementById('gardenPicture');
+const checkTemperatureRange = (countValue) => {
+  if (49 <= countValue && countValue < 50) {
+    document.body.style.backgroundColor = 'teal';
+    gardenPicture.textContent = '🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲';
+  } else if (50 <= countValue && countValue < 59) {
+    document.body.style.backgroundColor = 'green';
+    gardenPicture.textContent = '🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲';
+  } else if (60 <= countValue && countValue < 69) {
+    document.body.style.backgroundColor = 'yellow';
+    gardenPicture.textContent = '🌾🌾_🍃_🪨__🛤_🌾🌾🌾_🍃';
+  } else if (70 <= countValue && countValue < 79) {
+    document.body.style.backgroundColor = 'orange';
+    gardenPicture.textContent = '🌸🌿🌼__🌷🌻🌿_☘️🌱_🌻🌷';
+  } else if (80 <= countValue) {
+    document.body.style.backgroundColor = 'red';
+    gardenPicture.textContent = '🌵__🐍_🦂_🌵🌵__🐍_🏜_🦂';
+  }
+};
```

Comment:
The last `else if` can just be an `else`. Also, consider using classes instead of setting the color directly. This allows us to have all the styling information the the CSS file, and allows us easily have other type of styling be impacted by the temperature (for example if we decided we wanted the font to change depending on the temperature as well).